src/app/shared/components/delegated-acc-message/delegated-acc-message.component.ts

```
@@ -1,20 +1,39 @@
     });
+    this.setupNetworkWatcher();
+  }
+
+  setupNetworkWatcher(): void {
+    const networkWatcherEmitter = this.networkService.connectivityWatcher(new EventEmitter<boolean>());
+    this.isConnected$ = concat(this.networkService.isOnline(), networkWatcherEmitter.asObservable()).pipe(
+      shareReplay(1)
+    );
+  }
+
+  switchBack() {
```

ESLint

[error] Missing return type on function.

(@typescript-eslint/explicit-function-return-type)